Model/UrlInformation.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Model;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+class UrlInformation
+{
+    /**
+     * Decides whether the content is visible on sitemap.
+     *
+     * @var bool
+     */
+    private $visible;
+
+    /**
+     * @var string
+     */
+    private $loc;
+
+    /**
+     * @var \DateTime
+     */
+    private $lastmod;
+
+    /**
+     * @var string One of the official/allowed.
+     */
+    private $changeFreq;
+
+    /**
+     * @var float
+     */
+    private $priority;
+
+    /**
+     * @var array
+     */
+    private $allowedChangeFreqs = array('always', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'never');
+
+    /**
+     * @var string $label As a string to display the route i.e. in html views.
+     */
+    private $label;
+
+    /**
+     * @var array|AlternateLocale[]
+     */
+    private $alternateLocales;
+
+    public function __construct()
+    {
+        $this->alternateLocales = array();
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isVisible()
+    {
+        return $this->visible;
+    }
+
+    /**
+     * @param boolean $visible
+     */
+    public function setVisible($visible)
```

Comment:
yeah, i was wondering the same. just a constructor could be very easy. but then, we might have various voters or extractors in some situations, where it becomes really cumbersome to re-create the UrlInformation constantly just to update some data.

all in all, this could be an array with keys - its really just a value container. so imho setters are fine in this case.